docs/remote/containers.md

```diff
@@ -353,27 +353,40 @@ git config --global user.email "your.email@address"
 
 ### Using SSH keys
 
-However, in some cases you may be cloning your repository using SSH keys instead of a credential helper. SSH keys are not automatically mounted into the container since they can be used for things like accessing test or production servers and can pose a security risk.
+However, in some cases you may be cloning your repository using SSH keys instead of a credential helper. Simply mounting your local `~/.ssh` folder into the container works on macOS/Linux, but unfortunately this does not work on Windows due to the permissions Windows sets. The contents of the `.ssh` folder is not automatically copied into the container since some of the keys could be used for things like accessing test or production servers and could pose a security risk.
 
-However, it is easy enough to mount one or more keys into the container.
+ However, you can opt-into a cross-platform way of copying the contents `.ssh` folder into the container when it is created without modifying your image or Dockerfile.
 
-- When an **image** or **Dockerfile** is referenced in `devcontainer.json`, add the following to the `runArgs` property in this same file:
+- When an **image** or **Dockerfile** is referenced in `devcontainer.json`, add the following to this same file:
 
     ```json
-    "runArgs": [ "-v", "${env:HOME}${env:USERPROFILE}/.ssh:/root/.ssh" ]
+    // Mount your .ssh folder to /root/.ssh-localhost so we can copy its contents
+    "runArgs": [ "-v", "${env:HOME}${env:USERPROFILE}/.ssh:/root/.ssh-localhost:ro" ],
+    // Copy the contents to the correct location and set permissions
+    "postCreateCommand": "mkdir -p ~/.ssh && cp -r ~/.ssh-localhost/* ~/.ssh && chmod 700 ~/.ssh && chmod 600 ~/.ssh/*"
     ```
 
-- When a **Docker Compose** file is referenced, update ([or extend](/docs/remote/containers/containers.md#extending-your-docker-compose-file-for-development)) your `docker-compose.yml` with the following for the appropriate service:
+- When a **Docker Compose** file is referenced, first, update ([or extend](/docs/remote/containers/containers.md#extending-your-docker-compose-file-for-development)) your `docker-compose.yml` with the following for the appropriate service:
 
     ```yml
     version: '3'
     services:
       your-service-name-here:
         # ...
         volumes:
-          - ~/.ssh:~/.ssh
+          - ~/.ssh:~/.ssh-localhost:ro
     ```
 
+    And then add this to `devcontainer.json` to copy the keys to the correct spot with the right permissions:
+
+    ```json
+    "postCreateCommand": "mkdir -p ~/.ssh && cp -r ~/.ssh-localhost/* ~/.ssh && chmod 700 ~/.ssh && chmod 600 ~/.ssh/*"
+    ```
+
+You can modify the mount
```

Comment:
@Chuxel Is there more content to go here or can this sentence fragment be removed?

Reply:
Whoops!  Good catch.  Fixed.